variables.tf

```diff
@@ -22,7 +22,27 @@ variable "datadog_monitors" {
     threshold_windows   = map(any)
     thresholds          = map(any)
   }))
-  description = "List of Datadog monitor configurations"
+  description = "List of Datadog monitor configurations. See catalog for examples"
+}
+
+# https://registry.terraform.io/providers/DataDog/datadog/latest/docs/resources/synthetics_test
+variable "datadog_synthetics" {
+  default = {}
+  type = map(object({
+    name      = string
+    message   = string
+    type      = string
+    subtype   = string
+    status    = string
+    locations = list(string)
+    tags      = list(string)
+    assertions = list(object({
+      type     = string
+      operator = string
+      target   = any
+    }))
+  }))
+  description = "List of DataDog synthetic test configurations. See catalog for examples"
```

Comment:
```suggestion
  description = "List of Datadog synthetic test configurations. See catalog for examples"
```

Reply:
Ha, I've always written it this way... but you're right 👍